.github/workflows/codeql-build.yml

```diff
@@ -0,0 +1,57 @@
+name: "CodeQL Analysis"
+
+on:
+  push:
```

Comment:
See updated workflow files. This should now be 

``` yml
on:
  push:
    branches:
      - main
  pull_request:
```

Reply:
Updated to push on main branch.